packages/daemon/src/utils/queue.ts

```diff
@@ -0,0 +1,3 @@
+import Queue from 'queue';
+
+export const pluginQueue = new Queue({ autostart: true, concurrency: 1 });
```

Comment:
Do we really need this to be a separate file?

Reply:
Or merge this into `emitter.ts` and rename to `utils.ts`.

Reply:
I'm +1 on this :)

Reply:
Updated.